### PR TITLE
airframe: Add Nemo

### PR DESCRIPTION
**Pull request by dgeorge83616**
NOT A PR - DO NOT MERGE

The changes are an attempt to add a new airframe to PX4 and they are indeed sufficient to select the new airframe via QGC. However, doing so causes the RC pilot console link to disconnect after the reboot. Reselecting Generic Tailsitter restores the RC link after a reboot.

These changes were sufficient to enable the airframe on 1.10.2, but 1.11.3 presented other issues that are not manifest on `master`. Unfortunately, `master` has this other unexpected side effect of disabling the RC link.

The build target on master is `cubepilot_cubeorange_console`; on 1.10.2, it was `px4_fmu-v3`.

The custom mixer does rely upon px4io and I have also tried using [px4io lobotomy](https://github.com/PX4/PX4-Autopilot/pull/16444) but it makes no difference wrt this particular issue.

Please advise.

I'll promptly remove this PR upon resolution, or request.

### CODE REVIEW

File: ROMFS/px4fmu_common/init.d/airframes/13021_nemo_vtol [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+# @output MAIN5 Left Throttle
+# @output MAIN6 Right Throttle
+#
+# @output AUX1 Right Elevon
+# @output AUX2 Left Elevon
+#
+# @maintainer Christian Lopez <Christian@anduril.com>
+#
+
+sh /etc/init.d/rc.vtol_defaults
```
Comment (Jaeyoung-Lim) [inline ROMFS/px4fmu_common/init.d/airframes/13021_nemo_vtol:23]:
```suggestion
. ${R}etc/init.d/rc.vtol_defaults
```

**Review by Jaeyoung-Lim — CHANGES_REQUESTED** [review]
Your problem seems to be caused by a change on how the rcS is being called

**Comment by dgeorge83616** [comment]
> Your problem is caused by a change on how the rcS is being called

Thank you!

Now, does not boot far enough for QGC to connect, but that may be resolved by the px4io lobo).

**Comment by dgeorge83616** [comment]
> > Your problem is caused by a change on how the rcS is being called
> 
> Thank you!
> 
> Now, does not boot far enough for QGC to connect, but that may be resolved by the px4io lobo).

Oops, no, it doesn't boot when built with the [px4io lobo](https://github.com/PX4/PX4-Autopilot/pull/16444) latest commit cherry-picked. Investigating.

**Comment by dgeorge83616** [comment]
> > > Your problem is caused by a change on how the rcS is being called
> > 
> > 
> > Thank you!
> > Now, does not boot far enough for QGC to connect, but that may be resolved by the px4io lobo).
> 
> Oops, no, it doesn't boot when built with the [px4io lobo](https://github.com/PX4/PX4-Autopilot/pull/16444) latest commit cherry-picked. Investigating.

After removing the px4io lobo commit, the target boots and still no radio (no RSSI).

**Comment by LorenzMeier** [comment]
No RSSI doesn't necessarily mean you have no RC. You also might be using an RC system that doesn't provide RSSI.

Are the channel indicators moving in QGroundControl?

**Pull request closed without merging**